boardgame-ui/src/components/event/EventAddGame.vue

```diff
@@ -0,0 +1,292 @@
+<template>
+  <div class="wrapper">
+    <h1 class="title">{{$t('add-game.title')}}</h1>
+    <form @submit.prevent="save()">
+      <b-field
+        :label="$t('add-game.board-game.label')"
+        :type="{'is-danger': errors.has('boardGame')}"
+        :message="errors.first('boardGame')"
+      >
+        <b-autocomplete
+          v-model="searchString"
+          :data="filteredBoardGames"
+          field="name"
+          icon="search"
+          @select="selectBoardGame"
+          name="boardGame"
+          :data-vv-as="$t('add-game.board-game.label')"
+          v-validate="'required'"
+        >
+          <template slot="empty">{{$t('add-game.board-game.no-result')}}</template>
+        </b-autocomplete>
+      </b-field>
+      <div class="columns">
+        <div class="column">
+          <b-field :label="$t('add-game.ranking-method.label')">
+            <b-select v-model="game.ranking_method" expanded>
```

Comment:
I have this
![Screenshot 2019-04-16 at 23 31 56](https://user-images.githubusercontent.com/5320536/56245498-d7e9c880-609f-11e9-984c-219b06da9fe5.png)

Should we initialise game.ranking_method then check its value?

Also, should we use a placeholder?
placeholder="Select a ranking method"

Reply:
Oops, probably a regression due to the possibility to edit, thanks for noticing! I think I will mask the form until the initialization is finished, thus avoiding this error. Therefore, the ranking method will be initialized to "Ranking" and there will be no need for placeholder/front end validation. Ok for you ?

Reply:
Awesome!